src/storage/storage.cc

```diff
@@ -769,6 +775,65 @@ rocksdb::Status Storage::FlushScripts(engine::Context &ctx, const rocksdb::Write
   return Write(ctx, options, batch->GetWriteBatch());
 }
 
+rocksdb::Status Storage::IngestSST(const std::string &sst_dir, int* files_loaded) {
+  std::vector<std::string> sst_files;
+  DIR *dir = opendir(sst_dir.c_str());
+  if (!dir) {
+    return rocksdb::Status::IOError("Failed to open directory " + sst_dir);
+  }
+
+  struct dirent *entry;
+  while ((entry = readdir(dir)) != nullptr) {
+    std::string filename = entry->d_name;
+    if (filename.length() >= 4 && filename.substr(filename.length() - 4) == ".sst") {
+      sst_files.push_back(sst_dir + "/" + filename);
+    }
+  }
+  closedir(dir);
+
+  if (sst_files.empty()) {
+    LOG(WARNING) << "No SST files found in " << sst_dir;
+    return rocksdb::Status::OK();
+  }
+
+  std::vector<std::string> default_files;
+  std::vector<std::string> metadata_files;
+
+  // Sort files into appropriate vectors based on filename
+  for (const auto &file : sst_files) {
+    if (file.find("metadata") != std::string::npos) {
+      metadata_files.push_back(file);
+    } else {
+      default_files.push_back(file);
+    }
+  }
```

Comment:
Seems too ad-hoc here? Why not just use column family name instead of hard-coded string here?

Reply:
@PragmaTwice I see the following CF defined:
```
enum class ColumnFamilyID : uint32_t {
  PrimarySubkey = 0,
  Metadata,
  SecondarySubkey,
  PubSub,
  Propagate,
  Stream,
  Search,
};
```
from a quick look at this we could hardcode `default` and search in the folder for all the files that matches those CF? honestly i don't think all CF make sense, eager to hear your opinion.

Reply:
@ltagliamonte-dd The SST file writer will record the column family if it's assigned, so I think we could find the column family through the SST table properties when loading.

Please refer to https://github.com/facebook/rocksdb/blob/743a02d6f68811bed8c5f3ce5d7e6d9b6bc18783/table/sst_file_writer.cc#L396

Reply:
You can get CF names here: https://github.com/apache/kvrocks/blob/unstable/src/storage/storage.h#L197

Reply:
addressed this via 0ca11358751c7748ca4a0d5228ec4e620bc609aa
please let me know if you have any other feedbacks